do-app-platform-op-cli/README.md

```diff
@@ -1,7 +1,9 @@
-# [Beta] Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
+# Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
 
 This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and optionally the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/) (for Mac and Linux users).
 
+Alternatively, you can deploy the [1Password SCIM Bridge to DigitalOcean's App Platform Container-as-a-Service environment using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/) instead of the command line. 
```

Comment:
"instead" of is implied by "alternatively". I also cleaned up the linked text a bit, and I think in this case it's okay for "container" to not be plural since it doesn't read well. I'm tempted to remove it from this link, though, since it's not explained in this article and folks may not have context to know what it means. So if you agree, there's a suggestion without it below:

```suggestion
Alternatively, you can [deploy 1Password SCIM Bridge to DigitalOcean's App Platform container as a service environment using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/).
```

without CaaS:

```suggestion
Alternatively, you can [deploy 1Password SCIM Bridge to DigitalOcean App Platform using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/).
```

Reply:
I prefer the second option here (just say App Platform).